Source: John Kallimanis
Upstream: Not applied. Unlikely to be accepted. No active development.
Reason: Fix the build system for multiarch, especially for prefixed binutils/tools.
diff -rupN xephem-3.7.6/GUI/xephem/Makefile xephem-3.7.6-new/GUI/xephem/Makefile
--- xephem-3.7.6/GUI/xephem/Makefile	2013-01-20 03:42:02.000000000 +0000
+++ xephem-3.7.6-new/GUI/xephem/Makefile	2015-06-22 18:45:40.837507229 +0000
@@ -32,7 +32,7 @@ endif
 
 # for linux and Apple OS X
 # some systems now use just /usr/X11 (not /usr/X11R6)
-CC = gcc
+# CC = gcc
 CLDFLAGS = -g
 CFLAGS = $(LIBINC) $(CLDFLAGS) -O2 -Wall -I$(MOTIFI) -I/usr/X11R6/include
 LDFLAGS = $(LIBLNK) $(CLDFLAGS) -L$(MOTIFL) -L/usr/X11R6/lib
diff -rupN xephem-3.7.6/libastro/Makefile xephem-3.7.6-new/libastro/Makefile
--- xephem-3.7.6/libastro/Makefile	2006-02-24 03:37:31.000000000 +0000
+++ xephem-3.7.6-new/libastro/Makefile	2015-06-22 18:59:37.867180361 +0000
@@ -4,7 +4,7 @@
 # compiler and flags
 
 # gcc
-CC = gcc
+# CC = gcc
 CFLAGS= -O2 -Wall
 
 # macosx universal binary
@@ -83,8 +83,8 @@ OBJS =			\
 	vsop87_data.o
 
 libastro.a: $(HS) $(OBJS)
-	ar rv $@ $(OBJS)
-	ranlib $@
+	$(AR) rv $@ $(OBJS)
+	$(RANLIB) $@
 
 libastro.so: $(HS) $(OBJS)
 	$(CC) -shared -o $@ $(OBJS)
diff -rupN xephem-3.7.6/libip/Makefile xephem-3.7.6-new/libip/Makefile
--- xephem-3.7.6/libip/Makefile	2009-01-05 20:53:46.000000000 +0000
+++ xephem-3.7.6-new/libip/Makefile	2015-06-22 19:18:44.060965071 +0000
@@ -2,7 +2,7 @@
 # (C) 2001 Elwood Charles Downey
 
 # gcc
-CC = gcc
+# CC = gcc
 CFLAGS= -I../libastro -O2 -Wall
 
 # macosx universal binary
@@ -36,8 +36,8 @@ OBJS =	\
 HS = ip.h fsmatch.h
 
 libip.a: $(HS)	$(OBJS)
-	ar rv $@ $(OBJS)
-	ranlib $@
+	$(AR) rv $@ $(OBJS)
+	$(RANLIB) $@
 
 libip.so:    $(OBJS)
 	gcc -shared -o $@ $(OBJS)
diff -rupN xephem-3.7.6/libjpegd/Makefile xephem-3.7.6-new/libjpegd/Makefile
--- xephem-3.7.6/libjpegd/Makefile	2006-02-24 03:37:32.000000000 +0000
+++ xephem-3.7.6-new/libjpegd/Makefile	2015-06-22 19:04:14.283854566 +0000
@@ -3,7 +3,7 @@
 # compiler and flags
 
 # gcc
-CC = gcc
+# CC = gcc
 CFLAGS= -O2 -Wall
 
 # macosx universal binary
@@ -62,8 +62,8 @@ OBJS = \
 	jutils.o
 
 libjpegd.a: $(HS) $(OBJS)
-	ar r $@ $(OBJS)
-	ranlib $@
+	$(AR) r $@ $(OBJS)
+	$(RANLIB) $@
 
 clobber:
 	touch x.o x.a
diff -rupN xephem-3.7.6/liblilxml/Makefile xephem-3.7.6-new/liblilxml/Makefile
--- xephem-3.7.6/liblilxml/Makefile	2006-02-24 03:37:32.000000000 +0000
+++ xephem-3.7.6-new/liblilxml/Makefile	2015-06-22 19:08:54.384996948 +0000
@@ -4,7 +4,7 @@
 # compiler and flags
 
 # gcc
-CC = gcc
+# CC = gcc
 CFLAGS= -O2 -Wall
 
 # macosx universal binary
@@ -27,8 +27,8 @@ HS = lilxml.h
 OBJS = lilxml.o base64.o
 
 liblilxml.a: $(HS) $(OBJS)
-	ar r $@ $(OBJS)
-	ranlib $@
+	$(AR) r $@ $(OBJS)
+	$(RANLIB) $@
 
 liltest: liltest.o liblilxml.a
 	$(CC) $(LDFLAGS) -o liltest liltest.o liblilxml.a
diff -rupN xephem-3.7.6/libpng/Makefile xephem-3.7.6-new/libpng/Makefile
--- xephem-3.7.6/libpng/Makefile	2006-02-24 03:36:49.000000000 +0000
+++ xephem-3.7.6-new/libpng/Makefile	2015-06-22 19:20:18.903513730 +0000
@@ -8,10 +8,10 @@ ZLIBINC = ../libz
 ZLIBLIB = ../libz
 
 # Compiler, linker, lib and other tools
-CC = gcc
+# CC = gcc
 LD = $(CC)
-AR = ar rcs
-RANLIB = ranlib
+#AR = ar rcs
+#RANLIB = ranlib
 RM = rm -f
 
 CDEBUG = -g -DPNG_DEBUG=5
@@ -38,7 +38,7 @@ OBJS = png$(O) pngerror$(O) pngget$(O) p
 all: libpng$(A)
 
 libpng$(A): $(OBJS)
-	$(AR) $@ $(OBJS)
+	$(AR) rv $@ $(OBJS)
 	$(RANLIB) $@
 
 clean:
diff -rupN xephem-3.7.6/libz/Makefile xephem-3.7.6-new/libz/Makefile
--- xephem-3.7.6/libz/Makefile	2006-02-24 03:37:11.000000000 +0000
+++ xephem-3.7.6-new/libz/Makefile	2015-06-22 19:06:51.223548568 +0000
@@ -17,11 +17,11 @@ OBJS =	adler32.o	\
 	inffast.o
 
 libz.a: $(OBJS)
-	ar rc libz.a $(OBJS)
-	-ranlib libz.a
+	$(AR) rc libz.a $(OBJS)
+	$(RANLIB) libz.a
 
 testzlib: testzlib.o libz.a
-	cc $(LDFLAGS) -o testzlib testzlib.o libz.a
+	$(CC) $(LDFLAGS) -o testzlib testzlib.o libz.a
 
 clean:
 	rm -f *.o *.a testzlib
